### ISSUE TITLE
Add turbo ranked

### ISSUE THREAD

**Issue by Redo02** [ValveSoftware/Dota2-Gameplay#13345]
### Description

almost every people. playin turbo for score rigth?
since we cant increse score now. cuz of auto report counter.
they must add turbo ranked so i can LOSE SCORE PROPERLY
losing score for nothing is worthless
add turbo ranked then i can PROPERLY LOSE MY SCORE
INCREASING SCORE IS IMPOSSIBLE
1 WIN 1 SCORE NEED TO 3000 WIN.
ONLY WIN COUNT IF YOU LOSE YOU WILL GET REPORTED. EVEN U DIDNT TROLL AFK PING THROW ETC.
AFTER YOUR 2999 WIN DONT LOSE 3000TH GAME IT WILL COST 2999 SCORE. TO LOSE
THANK YOU FOR ADDING TURBO RANKED. I WANT RUIN PROPERLY. I DONT WANT PAY MY SCORE FOR NOTHING
and nothing left to pay

**Comment by PlumbusHo** [comment]
Yes! I usually only play turbo now and this would maybe attract new people to watch streams more often as the game are quicker and you don't have to watch an hour minimum. All about keeping people interested, so if the game is more fast paced, new viewers will be drawn in more!

**Comment by PlumbusHo** [comment]
Yes! I usually only play turbo now and this would maybe attract new people to watch streams more often as the game are quicker and you don't have to watch an hour minimum. All about keeping people interested, so if the game is more fast paced, new viewers will be drawn in more!

**Comment by TobyFlendersonFromHR** [comment]
low iq take imo

**Comment by PlumbusHo** [comment]
> low iq take imo

Thank for commenting, we value your opinion, in fact we would like someone like you to run things around here. A place you can home, a place where you can grow old and nerture you're true self. That's right not nerture your true self, but nerture your are true self. Someone who has the IQ to use grammar correctly, someone who's IQ, sorry "iq" is above all others, someone who need to save precious time by not using capital letters. This is the true form of intellegence.
Thank you. Xx <3
